I could use a helping hand. I am not sure what's the problem.

```
❯ make check
python3 tools/rstlint.py -i tools -i ./venv -i README.rst
No problems found.
python3 tools/rstlint.py ../Misc/NEWS.d/next/
[2] ../Misc/NEWS.d/next/Documentation/2021-03-02-12-55-34.bpo-43354.ezZYkx.rst:1: default role used
1 problem with severity 2 found.
Makefile:204: recipe for target 'check' failed
make: *** [check] Error 1
```

For me my snippet looks like valid RST.

--- a/Misc/NEWS.d/next/Documentation/2021-03-02-12-55-34.bpo-43354.ezZYkx.rst
+++ b/Misc/NEWS.d/next/Documentation/2021-03-02-12-55-34.bpo-43354.ezZYkx.rst
@@ -1,1 +1,1 @@
-Fix type documentation for `Fault.faultCode`; the type has to be `int` instead of `str`.
+Fix type documentation for ``Fault.faultCode``; the type has to be ``int`` instead of ``str``.

